### DISCUSSION TITLE
[Atlantis 5.1.0] Components are shown incorrectly with PrimeVue 3.35.0

### DISCUSSION THREAD

**Discussion by nk2048** [primefaces/community#182] [PrimeVue Templates] [answered]
I've noticed that the components SplitButton, SelectButton, SpinnerButton are shown incorrectly with PrimeVue 3.35.0.

**Appearance with PrimeVue 3.30.2**
![splitButton330](https://github.com/primefaces/community/assets/57187499/b0664b50-7727-49bd-95e9-1e8a3aa089d0)
![selectButton330](https://github.com/primefaces/community/assets/57187499/68067088-3fea-4407-b714-9e6cbd278f7c)
![spinnerButtons330](https://github.com/primefaces/community/assets/57187499/b86978db-beac-4a73-b972-a3c4a948c69c)

**Appearance with PrimeVue 3.35.0**
![splitButton335](https://github.com/primefaces/community/assets/57187499/b1bfd674-d1ce-4866-ba40-da378a55fc3a)
![selectButton335](https://github.com/primefaces/community/assets/57187499/e4b3e1b7-74d7-45a8-b9b0-aeee8ed166e4)
![spinnerButtons335](https://github.com/primefaces/community/assets/57187499/a8566dcd-c1fc-415e-968f-b20a9b7b4c69)

These components are used in our project and we would like to update PrimeVue to 3.35.0. Because of the above mentioned issue with the components, it cannot happen. When are you planning to adapt the template Atlantis to PrimeVue 3.35.0?
Thank you in advance.

**Comment by sezisfurkan** [comment, marked as answer]
Hi,
The issue has been fixed, and we will be publishing a new version soon, where we have corrected the errors to ensure a smooth release.

**Reply by fotisskal** [reply]
hello @sezisfurkan I am using primevue 3.44.0 but still facing the above issue as @nk2048 .

**Reply by fotisskal** [reply]
@sezisfurkan when are you planning to release this new version with the fix?

**Reply by sezisfurkan** [reply]
@fotisskal Sorry for the late response. It has been updated and uploaded to the store. You can download the updated version from the store.

**Reply by fotisskal** [reply]
hello @sezisfurkan thank you for your response. But version 5.2.0 seems to have serious issues with the css. I just downloaded it, ran 'npm install', then 'npm run dev' and the styling is weird.

**Reply by nk2048** [reply]
@fotisskal The updated version [5.2.0] has no css files in the public folder. Create them yourself using the following command
`sass public` 
and then run the template 
`npm run dev`.

**Reply by fotisskal** [reply]
@nk2048 thank you very much!
